### PR TITLE
do not show duplicate requests

### PR DESCRIPTION
**Pull request by morsdyce**

### CODE REVIEW

File: lib/ui/components/captured-requests.js [hunk 1]
```diff
@@ -82,10 +83,19 @@ export class CapturedRequests extends React.Component {
       : false;
   }
 
+  _getUniqueRequests(requests) {
+    return Uniqe(requests, false, (request) => {
```
Comment (borisd) [inline lib/ui/components/captured-requests.js]: One line?

File: lib/ui/components/captured-requests.js [hunk 2]
```diff
@@ -82,10 +83,19 @@ export class CapturedRequests extends React.Component {
...
+      return request.requestHash;
+    });
+  }
+
   _requestsList() {
+
+    console.log(API.capturedRequests);
```
Comment (borisd) [inline lib/ui/components/captured-requests.js]: Are you guys leaving console.logs on purpose?

File: lib/ui/components/captured-requests.js (hunk 1 above)
Comment (morsdyce) [inline lib/ui/components/captured-requests.js]: Thanks, forgot to refactor. Updated

File: lib/ui/components/captured-requests.js (hunk 2 above)
Comment (morsdyce) [inline lib/ui/components/captured-requests.js]: Nope, just forgot them in. Updated

**Pull request merged**